styles.css

```diff
@@ -98,19 +97,36 @@ h2 {
 }
 
 .socials > ul > li {
-  width: 2rem;
   list-style: none;
   padding: 4px;
 }
 
+.project > img {
+  border-radius: 8px;
+}
+
+.container > img {
+  width: 77vw;
+}
+
+.dropdown-content > a > img {
+  padding: 12px;
+}
+
+.socials > ul > li > a > img {
+  width: 5vw;
+}
+
 .container {
   display: grid;
-  padding: 16px;
+  padding: 1rem;
   background-color: #fff;
   border-radius: 16px;
   padding-bottom: 0;
   flex-direction: column;
   height: auto;
+  align-items: center;
+  justify-content: center;
 }
```

Comment:
- Currently, your design is not replicating the Figma design. Kindly fix the add the necessary margins or paddings

Your design
![image](https://github.com/agneta1991/Portfolio/assets/89091822/fcff5004-46f7-4572-ae6a-6f2e5ba76e02)

Figma design
![image](https://github.com/agneta1991/Portfolio/assets/89091822/cb553902-422b-4164-95b8-e444078f5711)